MH4U Dex/CoreDataController.h

```diff
@@ -18,6 +18,8 @@
 - (void)loadRegionData;
 - (BOOL)attemptSaveContext;
 
+- (NSManagedObject *)getUniqueEntityWithEntityName:(NSString *)entityName withPredicate:(NSPredicate *)predicate;
```

Comment:
This (and, actually, most if not all methods exposed in headers) should have a documenting comment.